modules/vorbis/audio_stream_ogg_vorbis.cpp

```diff
@@ -125,7 +126,7 @@ int AudioStreamPlaybackOggVorbis::_mix_internal(AudioFrame *p_buffer, int p_fram
 		if (!have_packets_left && !have_samples_left) {
 			// Actual end of file!
 			bool is_not_empty = mixed > 0 || vorbis_stream->get_length() > 0;
-			if (vorbis_stream->loop && is_not_empty) {
+			if (looping && is_not_empty) {
```

Comment:
```suggestion
			if (beat_loop && is_not_empty) {
```
Though renaming the variable to `use_loop` like in MP3 would be more clear.